src/graph/ShowExecutor.cpp

```diff
@@ -377,22 +386,25 @@ void ShowExecutor::showEdges() {
             return;
         }
 
+        std::unordered_set<TagID> edges;
         auto value = std::move(resp).value();
         resp_ = std::make_unique<cpp2::ExecutionResponse>();
         std::vector<cpp2::RowValue> rows;
         std::vector<std::string> header{"ID", "Name"};
         resp_->set_column_names(std::move(header));
 
-        std::map<nebula::cpp2::EdgeType, std::string> edgeItems;
         for (auto &edge : value) {
-            edgeItems.emplace(edge.get_edge_type(), edge.get_edge_name());
-        }
+            auto edgeType = edge.get_edge_type();
+            auto iter = edges.find(edgeType);
+            if (iter != edges.end()) {
+                continue;
+            }
 
-        for (auto &item : edgeItems) {
+            edges.emplace(edgeType);
             std::vector<cpp2::ColumnValue> row;
             row.resize(2);
-            row[0].set_integer(item.first);
-            row[1].set_str(item.second);
+            row[0].set_integer(edge.get_edge_type());
+            row[1].set_str(std::move(edge.get_edge_name()));
```

Comment:
std::move(edge).get_edge_name() ?

Reply:
This is move the string instance from edge.get_edge_name() I don't know why move the edge instance and get it's name